lock-impl/src/main/java/com/palantir/lock/impl/LegacyTimelockService.java

```diff
@@ -123,6 +125,21 @@ public StartIdentifiedAtlasDbTransactionResponse startIdentifiedAtlasDbTransacti
         }
     }
 
+    @Override
+    public StartIdentifiedAtlasDbTransactionResponseBatch startIdentifiedAtlasDbTransactionsBatch(int count) {
```

Comment:
Only batched method

Reply:
Oh wait this is legacy; @jeremyk-91 this is mostly unused now right so don't need to worry about perf?

Reply:
Fine by me: this is largely used in embedded contexts. There is large internal product, but in any case the algorithm here is not really worse than what people will probably end up writing anyway